tests/inputFields.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { test, expect } from '@playwright/test'
+
+test.describe('Input fields', async () => {
+	test.beforeEach(async ({ page }) => {
+		await page.goto('/')
+		await page.getByText('Pet Types').click()
+		await expect(page.getByRole('heading', { name: 'Pet Types' })).toBeVisible()
+	})
+
+	test('Update pet type', async ({ page }) => {
+		// Test data
+		const originalPetType = 'cat'
+		const newPetType = 'rabbit'
+
+		// Locators - Pet Types table page
+		const firstEditButton = page.getByRole('button', { name: 'Edit' }).first()
+		const firstPetTypeInput = page.locator('[name="pettype_name"]').first()
+		// Locators - Edit Pet Types page
+		const editPetTypeHeader = page.getByRole('heading', { name: 'Edit Pet Type' })
+		const petTypeInput = page.locator('#name')
+		const petTypeUpdateButton = page.getByRole('button', { name: 'Update' })
+
+		// Test steps
+		await firstEditButton.click()
+		await expect(editPetTypeHeader).toBeVisible()
+
+		await petTypeInput.click()
+		await petTypeInput.fill(newPetType)
+		await petTypeUpdateButton.click()
+		let inputValue = await firstPetTypeInput.inputValue()
+		expect(inputValue).toBe(newPetType)
```

Comment:
These two lines can be replaced with a single line Locator Assertion.
Also, try to avoid using toBe() assetion, always prioritize Locator Assertions vs Generic Assertion.
Read useful articles here: https://www.bondaracademy.com/blog/how-to-use-playwright-expect-assertions
And here: https://www.bondaracademy.com/blog/tobe-vs-toequal-assertion-playwright